templater/templater.go

```diff
@@ -169,6 +170,14 @@ func templateFuncs(rs *context.ResourceSet) template.FuncMap {
 		return string(b)
 	}
 	m["passLookup"] = GetFromPass
+	m["gitHEAD"] = func() (string, error) {
+			out, err := exec.Command("sh", "-c", "git rev-parse HEAD").Output()
```

Comment:
Any reason for executing it inside of a shell instead of going `Command("git", "rev-parse", "HEAD")` directly?

Reply:
Nope, makes more sense to use exec.Command directly in fact. Thnx